google/auth/aio/transport/aiohttp.py

```diff
@@ -14,3 +14,103 @@
 
 """Transport adapter for Asynchronous HTTP Requests.
 """
+
+import asyncio
+
+try:
+    import aiohttp
+except ImportError as caught_exc:  # pragma: NO COVER
+    raise ImportError(
+        "The aiohttp library is not installed from please install the aiohttp package to use the aiohttp transport."
+    ) from caught_exc
+from google.auth import exceptions
+
+_DEFAULT_TIMEOUT = 180  # in seconds
+
+
+class Request:
```

Comment:
Should this inherit from the abstract class?

Reply:
addressed.